test/gtest/ucp/test_ucp_rma_mt.cc

```diff
@@ -78,13 +78,21 @@ UCS_TEST_P(test_ucp_rma_mt, put_get) {
     st = ucp_rkey_pack(receiver().ucph(), memh, &rkey_buffer, &rkey_buffer_size);
     ASSERT_UCS_OK(st);
 
-    ucp_rkey_h *rkey;
+    std::vector<ucp_rkey_h> rkey;
+    rkey.resize(MT_TEST_NUM_THREADS);
 
-    rkey = (ucp_rkey_h *)malloc(sizeof(ucp_rkey_h) * sender().get_num_workers());
-    for (i = 0; i < sender().get_num_workers(); i++) {
-        st = ucp_ep_rkey_unpack(sender().ep(i), rkey_buffer, &rkey[i]);
+    /* test parallel rkey unpack */
+#if _OPENMP && ENABLE_MT
+#pragma omp parallel for
+    for (i = 0; i < MT_TEST_NUM_THREADS; i++) {
+        int worker_index = 0;
+        if (GetParam().thread_type == MULTI_THREAD_CONTEXT) {
+            worker_index = i;
+        }
+        st = ucp_ep_rkey_unpack(sender().ep(worker_index), rkey_buffer, &rkey[i]);
```

Comment:
minor: better to use local `ucs_status_t status` (or whatever name) variable here
```
for (...) {
    ...
    ucs_status_t status = ucp_ep_rkey_unpack(sender().ep(worker_index), rkey_buffer, &rkey[i]);
    ASSERT_UCS_OK(status);
}
```
or make it private:
```
#pragma omp parallel for private(st)
for (...) {
    ...
}
```